### DISCUSSION TITLE
Toonz_Level nodes using color's name.

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6563] [Ideas]
### Is your feature request related to a problem? Please describe

Sometimes it is difficult to know a color only by its index, especially when there are more than a dozen in a single level (when there are several levels it can be a headache) I know you can use the Style Picker, but I think it is a process quite slow, and doesn't always work, such as when you collapse a column into a sub-scene.

### Describe the solution you'd like

A solution that occurs to me is the possibility of being able to insert, in addition to the number, the name of the color directly in each node, so that the task is easier. If two or more colors have the same name, the node will only recognize the first one (by index), to fix this problem.

![image](https://user-images.githubusercontent.com/74098734/208992616-768a596c-fc36-4f45-80d7-e737237aa513.png)

I don't know if there might be an issue, or I'm letting go of some contentious issue that might come up, but it's an idea that occurred to me because I work a lot with smart levels, and I think it would help.

**Comment by RodneyBaker** [comment]
Are you suggesting that Tahoma2D already has this feature?

**Comment by F-Burning** [comment]
Sorry, this was an error xD Title and post fixed.

**Comment by KaseyFarron** [comment]
Interesting idea! I also made a similar request by introducing a colour picker option (https://github.com/opentoonz/opentoonz/issues/3681) so we could pick from the canvas.

**Comment by F-Burning** [comment]
That's the better option @KaseyFarron, but since Opentoonz seems to have very specific code, I don't know if it would be able to achieve something like that, that's why I came up with the names 😅

Edit: By specific code I meant a very specific way of working.

**Comment by DarrenTAnims** [comment]
An alternative could be to show the palette control in the panel, so that you click on the actual colour swatch to include it.
